CONTRIBUTING.md

```diff
@@ -455,6 +455,34 @@ Some important commands to know about:
 
 This module can be used to mock module variables, environment variables etc. Check out the official [monkeypatch documentation](https://docs.pytest.org/en/6.2.x/monkeypatch.html).
 
+## How to add new modules to the API
+
+## Opening and moving between branches
+
+To open a new branch:
+
+```bash
+git branch branch_name
+```
+
+To move to a given branch:
+
+```bash
+git checkout branch_name
+```
+
```

Comment:
Add the next two lines:

The last two commands can be combined into one command:

```bash
git checkout -b branch_name develop
```

Reply:
Add this in line 473.